e2e/commands/patient-operations.ts

```diff
@@ -96,10 +98,64 @@ export const generateRandomPatient = async (api: APIRequestContext): Promise<Pat
 };
 
 export const getPatient = async (api: APIRequestContext, uuid: string): Promise<Patient> => {
-  const patientRes = await api.get(`patient/${uuid}?v=full`);
+  const patientRes = await api.get(`rest/v1/patient/${uuid}?v=full`);
   return await patientRes.json();
 };
 
 export const deletePatient = async (api: APIRequestContext, uuid: string) => {
-  await api.delete(`patient/${uuid}`, { data: {} });
+  await api.delete(`rest/v1/patient/${uuid}`, { data: {} });
+};
+
+export const createImmunizations = async (api: APIRequestContext, uuid: string, visit: string) => {
+  const immunizationData = {
+    resourceType: 'Immunization',
+    status: 'completed',
+    vaccineCode: {
+      coding: [
+        {
+          code: '783AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+          display: 'Hepatitis B vaccination',
+        },
+      ],
+    },
+    patient: {
+      type: 'Patient',
+      reference: `Patient/${uuid}`,
+    },
+    encounter: {
+      type: 'Encounter',
+      reference: `Encounter/${visit}`,
+    },
+    occurrenceDateTime: '2024-06-10T13:50:00.000Z',
+    expirationDate: '2052-06-29T18:30:00.000Z',
+    location: {
+      type: 'Location',
+      reference: `Location/${process.env.E2E_LOGIN_DEFAULT_LOCATION_UUID}`,
+    },
+    performer: [
+      {
+        actor: {
+          type: 'Practitioner',
+          reference: 'Practitioner/f39e57d8-1185-4199-8567-6f1eeb160f05',
```

Comment:
I had a chat with Ian, and it has nothing to do with the encounter but we suspect this is what causing issue as this UUID doesn't exist. Let me update the PR. 

These are the valid values: https://github.com/openmrs/openmrs-module-referencedemodata/blob/2.x/api/src/main/java/org/openmrs/module/referencedemodata/ReferenceDemoDataConstants.java